src/App.tsx

```diff
@@ -1,14 +1,45 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { wait } from './utils/fetchClient';
+import classNames from 'classnames';
+import { FilterTodos } from './components/FilterTodos';
+import { Status } from './types/Status';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [status, setStatus] = useState<Status>(Status.All);
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setErrorMessage('Unable to load todos');
+
+        return wait(3000).then(() => setErrorMessage(''));
```

Comment:
Create a function that will set an error and then remove it in 3 second, as you have to do it for all requests